apps/nuxt3-ssr/custom-build.js

```diff
@@ -17,16 +17,10 @@ fs.readdir(location, (err, files) => {
   }
 });
 
-var fingerPrint = Date.now().toString(36);
-fs.writeFileSync(
-  "./utils/fingerprint.js",
-  "export const hash = '" + fingerPrint + "';"
-);
-
 shell.exec(
-  `tailwindcss -c ./tailwind.config.cjs -i ${location}main.css -o ${location}styles.${fingerPrint}.css --minify`
+  `tailwindcss -c ./tailwind.config.cjs -i ${location}main.css -o ${location}styles.css --minify`
 );
 shell.exec(
-  `tailwindcss -c ./tailwind.config.umcg.cjs -i ${location}main.css -o ${location}styles.umcg.${fingerPrint}.css --minify`
```

Comment:
this breaks css cache busting, and results in styles not updating in production context

Reply:
I understand, but can you come up with another solution that doesn't take me multiples of 30sec of my life?

Reply:
E.g. we could do this only at release?

Reply:
No this needs to be done every time a (prod)build is run, it takes max a few milliseconds. Yes this is a work around that should be replaced, but this would require a deep dive into the vite-rollup-nuxt-tailwind combination. The problem here seems to be the gradle cache system, the gradle metal-model/api seems to be hard to configure